src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/components/SceneFunctionDetailsPanel/components/CodeContainer/CodeContainer.tsx

```diff
@@ -1,18 +1,19 @@
-import { AiSuggestionsPanel } from '@shared/components/AiSuggestionsPanel/AiSuggestionsPanel';
 import { displayError } from '@shared/domain/displayStatus';
 import { HttpClientError } from '@shared/infrastructure/http/HttpClientError';
 import React from 'react';
 
-import { FunctionDetails } from '../../types/FunctionDetails';
+import { FunctionDetails } from '../../domain/types/FunctionDetails';
+import { AiSuggestionsPanel } from '../AiSuggestionsPanel/AiSuggestionsPanel';
 import { useCodeContainer } from './domain/useCodeContainer';
 import { Code } from './ui/Code';
 
 type CodeContainerProps = {
+  dataSourceUid: string;
   functionDetails: FunctionDetails;
 };
 
-export function CodeContainer({ functionDetails }: CodeContainerProps) {
-  const { data, actions } = useCodeContainer(functionDetails);
+export function CodeContainer({ dataSourceUid, functionDetails }: CodeContainerProps) {
+  const { data, actions } = useCodeContainer(dataSourceUid, functionDetails);
```

Comment:
Not ideal, we should find a better solution, like instantiating the API clients in `GitHubContextProvider` instead of prop-drilling the data source UID. We should do it in the next PR.